app/grandchallenge/algorithms/views.py

```diff
@@ -335,6 +336,39 @@ def get_context_data(self, *args, **kwargs):
         return context
 
 
+class AlgorithmImageMarkDesired(
+    LoginRequiredMixin, ObjectPermissionRequiredMixin, View
+):
+    permission_required = "algorithms.change_algorithmimage"
```

Comment:
Should this be change algorithm, not change algorithm image?

Reply:
Hm, I figured since we're modifying a specific algorithm image here, it should be change algorithm image (in line with our algorithm image update view).

Reply:
I know, we could argue either way. I think it should be for the algorithm as ultimately you are changing which image is active for it. That involves changing multiple algorithm images, not just the one you're checking permissions on right now.